client/scss/components/_help-block.scss

```
@@ -60,25 +60,16 @@
 }
 
 // Media for Windows High Contrast
-@media (forced-colors: $media-forced-colours) {
+@media (forced-colors: active) {
   .help-block {
-    forced-color-adjust: none;
-    border: 1px solid $system-color-link-text; // ensure visible separation in Windows High Contrast mode
-    background-color: transparent;
-    color: $color-white;
-
-    &:before {
-      color: currentColor;
-    }
+    border: 3px solid $system-color-link-text; // ensure visible separation in Windows High Contrast mode
```

I don’t think we want the help blocks to appear as links so to me it seems most appropriate to give them the same color as the text (so CanvasText) by default.

```suggestion
    border: 3px solid currentColor;
```